src/Uno.UI.RemoteControl/HotReload/ClientHotReloadProcessor.MetadataUpdate.cs

```diff
@@ -208,6 +208,16 @@ private static void ReplaceViewInstances(Func<FrameworkElement, bool> predicate)
 
 		public static void UpdateApplication(Type[] types)
 		{
+			var meta = (from t in types
+						let update = t.GetCustomAttribute<System.Runtime.CompilerServices.MetadataUpdateOriginalTypeAttribute>()
+						where update is not null
+						select (Type: t, OriginalType: update.OriginalType)).ToArray();
+
+			foreach (var t in meta)
+			{
+				TypeMappingHelper.RegisterMapping(t.Type, t.OriginalType);
+			}
```

Comment:
```suggestion
			foreach (var t in types)
			{
				if (t.GetCustomAttribute<System.Runtime.CompilerServices.MetadataUpdateOriginalTypeAttribute>() is { } update)
				{
					TypeMappingHelper.RegisterMapping(t, update.OriginalType);
				}
			}
```

or at least remove `.ToArray()`